checkpoint_ngfw/plugin.spec.yaml

```diff
@@ -692,6 +692,50 @@ actions:
         type: boolean
         required: true
 
+  set_threat_protection:
+    title: Set Threat Protection
+    description: Set threat protection action per profile
+    input:
+      name:
+        title: Name
+        description: Name of the protection to act on. e.g. Blaster Attacks
+        required: true
+        type: string
```

Comment:
We can start adding the `example:` idea for each input. Moving forward with this